### DISCUSSION TITLE
Support subfolder

### DISCUSSION THREAD

**Discussion by paulpach** [openupm/openupm#1233] [Q&A] [answered]
Unity now supports the package to be inside a subfolder of a repo.  For example,  in MirrorNG I am getting rid of the upm branch and instead use this path:

https://https://github.com/MirrorNG/MirrorNG.git?path=/Assets/Mirror

See unity documentation here:  https://docs.unity3d.com/Manual/upm-git.html#subfolder

[a comment by an automated account is not shown]

**Comment by favoyang** [comment, marked as answer]
It's been supported already. The build-pipelines just track Git tags, so as long as you have good Git tags created from your master branch we're good to go.

In your case,
- https://github.com/openupm/openupm/blob/master/data/packages/com.mirrorng.mirrorng.yml#L11 the tag with "-master" suffix is ignored, you may want to remove it (set to '') to match your Git tag naming pattern.
- I also notice that your https://github.com/MirrorNG/MirrorNG/blob/master/Assets/Mirror/package.json in the master branch has been reset to version 1.0.0. That is by intention or maybe a bug?

**Comment by favoyang** [comment]
@paulpach please see above.
